dapps/organizations/src/components/detail/identifications/notary/request/request.vue

```diff
@@ -275,12 +275,17 @@
 
```

Comment:
just talked with Sandra about the concept for this, the idea was to do it like this Screen. ...
<img width="784" alt="Screenshot 2019-08-28 at 16 06 33" src="https://user-images.githubusercontent.com/906623/63863090-f0eb7a00-c9ad-11e9-9e8c-d04353ecb804.png">

Reply:
Buy EVE link should route in the old profile buy screen ...

Reply:
adjusted.

German: 
![image](https://user-images.githubusercontent.com/763247/63923021-14fa9a00-ca46-11e9-90d7-6586a5d6709f.png)

English:
![image](https://user-images.githubusercontent.com/763247/63923128-496e5600-ca46-11e9-912b-6676f0972717.png)